packages/fxa-content-server/tests/functional/change_password.js

```diff
@@ -57,9 +57,6 @@ registerSuite('change_password', {
     email = TestHelpers.createEmail();
   },
 
-  afterEach: function() {
```

Comment:
It isn't obvious to me why this was removed?

Reply:
Because we do it in the beforeEach, and I think everywhere should do it in beforeEach. Doing it in afterEach and then beforeEach in the subsequent tests just slows things down.